OsmAnd/res/layout/fragment_edit_poi_advanced.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.core.widget.NestedScrollView
+<ScrollView
```

Comment:
We can't use ScrollView / NestedScrollView coudln't be used here cause it's already inside scrollview. Also we need to use recyclerview for tag/ values on advanced tabs